vpr/src/base/vpr_types.h

```diff
@@ -1412,6 +1412,40 @@ struct t_det_routing_arch {
     /// the CUSTOM switch block type. See comment at top of SRC/route/build_switchblocks.c
     std::vector<t_switchblock_inf> switchblocks;
 
+    /// Sub type and Fs are applied to pass tracks
```

Comment:
A bunch of these are only used with the tilable rr-graph generator. Should say that here; group them and mark that they only affect that rr-graph generator.

Reply:
I see you have a tileable section, but aren't sub_fs and switch_block_subtype also only for tileable rr-graphs?